### DISCUSSION TITLE
Z2M Notifications do not disappear.

### DISCUSSION THREAD

**Discussion by klixx23** [Koenkk/zigbee2mqtt#20436] [General]
Hello everyone,

the system messages no longer disappear in the Z2M web interface. What do I have to do to make the messages disappear again after a short time?

I am using Z2M version 1.34.0.

Many thanks in advance.

<img width="1177" alt="z2m_message" src="https://github.com/Koenkk/zigbee2mqtt/assets/121963300/4a145a33-c8a6-4e47-a7e0-5dd456a10e8f">

**Comment by Koenkk** [comment]
Will be fixed in the 1 January Z2M release.
